src/shims/env.rs

```diff
@@ -111,4 +112,31 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             Ok(-1)
         }
     }
+
+    fn getcwd(
+        &mut self,
+        buf_op: OpTy<'tcx, Tag>,
+        size_op: OpTy<'tcx, Tag>,
+    ) -> InterpResult<'tcx, Scalar<Tag>> {
+        let this = self.eval_context_mut();
+
+        let buf = this.read_scalar(buf_op)?.not_undef()?.assert_ptr();
```

Comment:
Use force ptr, it may be an integer address of a real allocation

Reply:
that might happen? ok